.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,26 @@
+name: leaderboard-backend CI
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  Golang-CI:
```

Comment:
Same naming question.

Reply:
renamed to `backend-test` _some_ name needs to exist otherwise the actions just don't kick off